src/modules/cmd_duration.rs

```diff
@@ -51,7 +51,7 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 }
 
 // Render the time into a nice human-readable string
-fn render_time(raw_millis: u128, show_millis: bool) -> String {
+pub fn render_time(raw_millis: u128, show_millis: bool) -> String {
```

Comment:
@matchai Should this be pulled into a shared file?

Reply:
Yeah, I think it would make sense to move into https://github.com/starship/starship/blob/master/src/utils.rs 😊